content/01.abstract.md

```diff
@@ -11,3 +11,21 @@ Publication with Manubot is continuous:
 When a manuscript's source changes, the rendered outputs are rebuilt and republished to a web page.
 Manubot automates bibliographic tasks by implementing citation by identifier, where users cite persistent identifiers (e.g. DOIs, PubMed IDs, ISBNs, URLs), whose metadata is then retrieved and converted to a user-specified style.
 Manubot modernizes publishing to align with the ideals of open science by making it transparent, reproducible, immediate, versioned, collaborative, and free of charge.
+
+## Author summary
+
+Traditionally, scholarly manuscripts, such as this one you are reading, have been written in private.
+This worked because the research was also performed in private by a predefined team of collaborators.
+But now the internet has enabled realtime open science, where project communication occurs online in a public venue and anyone is able to contribute.
+But how can dispersed teams of online contributors jointly prepare manuscripts to describe their work?
+
+Existing tools fail to scale and struggle to enable iterative refinement of proposed changes.
+Therefore, we created a system for writing manuscripts on GitHub, called Manubot.
+Manubot adopts the workflow from open source software development, which has enabled hundreds of contributors to simultaneously develop complex codebases such as Python and Linux, and applies it to open collaborative writing.
+
+In addition to collaboration at scale, Manubot addresses other shortcomings of current publishing tools.
+Specifically, all changes to a manuscript are tracked, enabling transparency and better attribution of credit.
+Manubot automates as many tasks as possible, including bibliography creation.
+Users cite persistent identifiers, such as DOIs, and then Manubot retrieves and formats the metadata into references.
+Furthermore, every version of a manuscript is automatically published and preserved with a permalink.
```

Comment:
`permalink` is correct, but `permanent URL` or `stable URL` would be more accessible.  Do we want to advertise these as "permanent" given potential stability issues with GitHub Pages?